SUPPORTED_LANGUAGES.md

```diff
@@ -4,6 +4,16 @@ The table below shows the full list of supported languages (and corresponding cl
 
 Languages that listed a **Package** below are 3rd party languages and are not bundled with the core library.  You can find their repositories by following the links.
 
+### Using 3rd Party Languages: 
```

Comment:
If anything this should link to the appropriate section of our README (where we already show how to do this). Also, just FYI, most 3rd party plugins include these instructions already in their own READMEs.